docs/arch/adr-04-merging-mainstream-to-specialist-topics.md

```diff
@@ -0,0 +1,27 @@
+# Merging of Mainstream Topics into Specialist Topics
+
+## Status
+
+Proposed
+
+## Context
+
+GOV.UK currently maintains 2 separate topic trees that represent a navigational structure for users. Mainstream Browse is the oldest and considered legacy. Specialist Topics are newer but need tidying up and do not power breadcrumbs.
+
+Previous attempts to merge the two stalled (since 2015).
+
+The newest tree, Taxonomy, does not affect navigation and is treated as a purely metadata type categorisation tree and is ignored by this work.
+
```

Comment:
Should we give a nod here to WHY we want to merge the trees. Ie state the problem we're trying to solve. Sam explained that this was because users can get siloed in MSB pages with no links to Spesh topics etc. Or is the fundamental problem the one experienced by our publishers, ie the consequences of tagging to a MSB or a specialist topic are confusing. Or is this a precursor for the wider longer term remit of the explore team to make it easier to find content on govuk by removing specialist topic pages?

Reply:
Added - thanks!